### DISCUSSION TITLE
Deep Shrink - anyone heard of it?

### DISCUSSION THREAD

**Discussion by lbeltrame** [vladmandic/sdnext#2721] [General]
Note, I'm not asking for inclusion in SD.Next, just discussing it. 

I noticed that this version of "hires fix" (https://gist.github.com/kohya-ss/3f774da220df102548093a7abc8538ed) is making the rounds in the Japanese community (and apparently it is implemented in ComfyUI). However, I don't know enough of the innards to understand what it does and, to be honest, why is it better than the currently available alternatives. 

Has anyone heard of it?

**Comment by vladmandic** [comment]
another nickname for that is soft-hires. i'm following the progress, but its not really stable just yet.

**Comment by lbeltrame** [comment]
Do you know why it's supposedly (I mean the adjective - I know nothing about it), "better" than the existing hires solutions?

**Reply by vladmandic** [reply]
idea behind it instead of just initializing noise at the start, it actively guides generation during forward pass so its more subtle in each step. similar concept how lora influences generation in real-time, not during init.
